prep-html-script-tag/index.html

```diff
@@ -7,8 +7,7 @@
     <script>
       console.log('I have added JS to a script tag in the HTML!');
     </script>
-    <script src="main.js"></script>
-    <script>
+    <script src="main.js">
       console.log('I have added script file via the src attribute!');
```

Comment:
It looks like you still have your `console.log` directly in-between your opening and closing `<script>` tag. You should have created a `main.js` file, and you need to move this `console.log` into that `main.js` file.